dma.c

```diff
@@ -131,13 +123,21 @@ set_from(struct queue *queue, const char *osender)
 	}
 
 	queue->sender = sender;
+
+	if(masquerade_config != NULL)
```

Comment:
maybe cleaner to move the conditional into free_masquerade_setting

Reply:
Okay.